python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -452,6 +452,10 @@ public static boolean isValidForProjectPythonVersion(List<String> validForPython
         symbols.add(new FunctionSymbolImpl(((SymbolsProtos.FunctionSymbol) descriptor), isInsideClass));
       }
       if (descriptor instanceof OverloadedFunctionSymbol) {
+        if (((OverloadedFunctionSymbol) descriptor).getDefinitionsList().size() < 2) {
+          LOG.error("Overloaded function symbols should have at least two definitions.");
```

Comment:
I think we shouldn't silently continue (as we do now). Instead I believe we should throw an Exception